### DISCUSSION TITLE
My reward was not received？why？

### DISCUSSION THREAD

**Discussion by jiajia4451** [Chia-Network/chia-blockchain#6510] [Q&A]
**I have not received the rewards for the following 3 height blocks. The height and our reward address are provided below.
My English is not very good, please understand.**

Click here to go to block of height 373555

Farmer reward | 0x7c52f81e3a9d3f688d439f94e652dc1064ffe8679e287b9c1d9f649795841928 | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 1.75
-- | -- | -- | --
🌱 Farmer reward | 0x551f38fc87af5378c5e58b62e064b094516f2bd5315515cdb4ce86af2cea5930 | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 0.25

and
Click here to go to block of height 368064

🌱 Farmer reward | 0x4a1286f8eb69f42b0278f3f1e959c38cc3255df9b0d53e63312e38b3d827aa3e | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 1.75
-- | -- | -- | --
🌱 Farmer reward | 0xb0e6d744a10ddb85c0dc52b3ad66474a743a7fb0e7e0af1ac9dd50bc235dd51d | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 0.25

and
Click here to go to block of height 364590

Farmer reward | 0x69e89e79d20c0c9a3f6f903ee18b52458140414964f5c79a4fecde9f070f9bf7 | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 1.75
-- | -- | -- | --
🌱 Farmer reward | 0x337c330c8c43e15bb7b97409ee6716fb4f508a43dd8a4e2b8b7f3424153e6682 | xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5 | 0.25

**Comment by jiajia4451** [comment]
**I also received a new reward today, and the past is normal.
Only these 3 high rewards were not received.**

**Reply by loppefaaret** [reply]
https://www.chiastatus.com/addresses/xch1vwrlqafr7ffhq2kl5nux8skgdfqvjv56mpavgfqdmxn8uflvk33sfja0h5
they were received, and send
